post/views.py

```
@@ -0,0 +1,260 @@
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+        except:
+            return Response(
+                {"detail": "User not found."}, status=status.HTTP_404_NOT_FOUND
+            )
+
+        ### 3 ###
+        is_liked = post.like_set.filter(user=author).count() > 0
+
+        ### 4 ###
+        if is_liked == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if is_liked:` for truth checks

Replace with `is_liked`

(E712)